Makefile

```diff
@@ -105,6 +105,10 @@ install: | prepare
 	fi
 	$(call message,Congratulations! You installed $(PROJECT_NAME)!)
 
+add\:additional-modules:
```

Comment:
I create new command because i do not know how i can use env variables in config CI.

Reply:
We don't need this additional command, you can use the `make drush en \$\(ADDITIONAL_MODULES\)` command in the CI.